muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/base/material-loader/fields/connect-field.tsx

```diff
@@ -573,6 +574,26 @@ export default class ConnectField extends React.Component<
           i18n={this.props.i18n}
           onFieldSavedStateChange={this.onFieldSavedStateChange.bind(this)}
         />
+        <span className="material-page__taskfield-header">
+          <span></span>
```

Comment:
Empty span?

Reply:
That span is usually reserved for taskfields that has title set but many cases there aren't titles. 

We still need that span element to position the instructions component to the right and it would have been unnecessary to create multiple modifiers to achieve the same thing without interfering the taskfield's alignment itself.